cpp/src/c_api/array.cpp

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-#include <c_api/array.hpp>
-#include <c_api/error.hpp>
-#include <c_api/resource_handle.hpp>
+#include "c_api/array.hpp"
+
```

Comment:
Why do we have an empty line here?

Reply:
There were a small number of cases like this in cudf, too. We suspected a clang-format bug was responsible.